### PR TITLE
added footer component

### PR DESCRIPTION
**Pull request by rkapoor10**
- added footer component
footer folder

- footer 
footer ss

Go to home page to check the UI

### CODE REVIEW

File: src/components/shared/footer/FooterComponent.jsx [hunk 1]
```diff
@@ -1,9 +1,69 @@
...
+            <li>
+              {" "}
+              <a href="./">Youtube</a>
+            </li>
+            <li>
+              {" "}
+              <a href="./">Instagram</a>
+            </li>
+            <li>
+              {" "}
```
Comment (JantuDeb) [inline src/components/shared/footer/FooterComponent.jsx]: Remove empty {}

**Review by harshitadaswani — APPROVED** [review]
The code structure looks good! I liked the way you have further broken it down into reusable components. Well done!

File: src/components/shared/footer/FooterComponent.jsx [hunk 2]
```diff
@@ -1,9 +1,69 @@
...
+        <div class="child-2 footer-middle-wrapper">
+          <div class="category-flex fw-bold gray">
+            <div class="category">
+              <a href="/products-page/products.html">Racquets |</a>
+            </div>
+            <div class="category">
+              <a href="/products-page/products.html">Shoes |</a>
+            </div>
+            <div class="category">
+              <a href="/products-page/products.html">Apparel</a>
```
Comment (JantuDeb) [inline src/components/shared/footer/FooterComponent.jsx]: product.html is not a correct path.

**Review by JantuDeb — APPROVED** [review]
Added a suggestion.

File: src/components/shared/footer/FooterComponent.jsx (hunk 1 above)
Comment (rkapoor10) [reply src/components/shared/footer/FooterComponent.jsx]: ok done

File: src/components/shared/footer/FooterComponent.jsx (hunk 2 above)
Comment (rkapoor10) [reply src/components/shared/footer/FooterComponent.jsx]: corrected it

**Pull request merged**